composer.json

```diff
@@ -34,6 +34,11 @@
             "Doctrine\\Tests": "vendor/doctrine/dbal/tests"
         }
     },
+    "config": {
```

Comment:
the config attribute is root-only so does not has any effect on projects using this library https://getcomposer.org/doc/04-schema.md#config